### ISSUE TITLE
6.2.3 Update Bash to version 5

### ISSUE THREAD

**Issue by kerempg** [k8-proxy/test-installation-documentation#41]
During Mac OS X testing,

For **6.2.3 Update Bash to version 5**
`brew install bash`

Following error is encountered:

Error: bash: no bottle available!
 You can try to install from source with:
 brew install --build-from-source bash
 Please note building from source is unsupported. You will encounter build
 failures with some formulae. If you experience any issues please create pull
 requests instead of asking for help on Homebrew's GitHub, Twitter or any other
 official channels.

when
`brew install --build-from-source bash`
is entered, it is successfully installed.

In summary, this step failed for me and brew install --build-from-source bash is alternate command
